internal/conf/zookeeper_test.go

```diff
@@ -0,0 +1,42 @@
+package conf
+
+import (
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/yaml.v3"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+)
+
+func ZkReadConfig(t *testing.T) *Config {
+	wd, _ := os.Getwd()
+	dir := wd[:strings.Index(wd, "internal")]
+	content, err := ioutil.ReadFile(dir + "conf/conf.yaml")
+	assert.Nil(t, err)
+	config := &Config{}
+	err = yaml.Unmarshal(content, config)
+	assert.Nil(t, err)
+	return config
+}
+
+func TestZkBuilderConfig(t *testing.T) {
+	config := ZkReadConfig(t)
+	assert.NotNil(t, config)
```

Comment:
I think it's more concise to get the config directly from a string, it's not clear to read from a file

Reply:
Great idea, the way you said it would be better.